examples/shipping-info-example/shipping-info-example.html

```diff
@@ -144,7 +143,7 @@ <h5 class="text-center">Response</h5>
 <!--Include script with VGS Collect form initialization-->
 <script>
   // VGS Collect form initialization
-  const form = VGSCollect.create('<vault-id>', function(state) {});
+  const form = VGSCollect.create('<vault-id>', 'sandbox', function(state) {});
```

Comment:
the better way - show customers all possible choose 
`'sandbox'` -> `<'sandbox' || 'live'>`